gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -542,6 +546,25 @@ message VehiclePosition {
   // This message/field is still experimental, and subject to change. It may be formally adopted in the future.
   repeated CarriageDetails multi_carriage_details = 11;
 
+  // The passenger crowd level for the vehicle or carriage.
+  // This field is still experimental, and subject to change. It may be formally adopted in the future.
+  // If multi_carriage_details is populated with per-carriage CrowdLevel,
+  // then this field should describe the entire vehicle with all carriages accepting passengers considered.
+  enum CrowdLevel {
+    // The vehicle or carriage does not have any crowd level data available at that time.
+    NO_DATA = 0;
```

Comment:
I was going to say let's use `NO_DATA_AVAILABLE` to match the other occupancy "no data" values. But we do have `ScheduleRelationship.NO_DATA`, so `OccupancyStatus` probably should have used this. So we could stick with `NO_DATA` here, especially if `OccupancyStatus` ends up being deprecated.

Reply:
FWIW I had originally used `NO_DATA_AVAILABLE` to match that in `OccupancyStatus` but the validator rejected it since `NO_DATA_AVAILABLE` has to be unique within VehiclePosition, not just CrowdLevel.

Reply:
Ah, right, I forgot about that 👍